### DISCUSSION TITLE
Connection and Channel Creation

### DISCUSSION THREAD

**Discussion by intisor** [rabbitmq/rabbitmq-dotnet-client#1747] [General]
### Describe the bug

 var factory = new ConnectionFactory() { HostName = "localhost" };
 _connection = factory.CreateConnection();
 its says it is only the CreateConnectionAsync thats available no CreateConnection()
 am i missing something

### Reproduction steps

1.
2.
3.
...

### Expected behavior

i expect the CreateConection() to be available

**Comment by michaelklishin** [comment]
@intisor please never ever use issues for questions in repositories that have Discussions enabled.

The code snippet you have shared is valid for `6.x` releases of this library. In 7.0, key operations have the `Async` suffix. The tutorials on rabbitmq.com were [updated accordingly](https://www.rabbitmq.com/tutorials/tutorial-one-dotnet#sending).

See [`v7-migration`](https://github.com/rabbitmq/rabbitmq-dotnet-client/blob/main/v7-MIGRATION.md) and 
https://github.com/rabbitmq/rabbitmq-dotnet-client/discussions/1720.

**Reply by intisor** [reply]
yh, thanks fixed it
